### PR TITLE
Adds submission history dev ready PDFs

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds dev-ready design PDFs for Submission History V2 / V3 (overlooked adding these after we had the handoff sync for #1290)

V2 supports #1609 
V3 supports #1610, #1612

### CODE REVIEW

**Comment by stevenino** [comment]
@reitermb is there documentation for the current submission portion?

**Comment by reitermb** [comment]
> @reitermb is there documentation for the current submission portion?

Yeah, that was on the list of past (v0/ATO) release spec we'd discussed not adding to the repo last week, but I did dig up the right Figma pages for it.

The Upload/Download section of https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=3588%3A800 is probably the simplest view of it—with https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=4178%3A1794 going into a lot more depth on various edge cases.

**Comment by stevenino** [comment]
> > @reitermb is there documentation for the current submission portion?
> 
> Yeah, that was on the list of past (v0/ATO) release spec we'd discussed not adding to the repo last week, but I did dig up the right Figma pages for it.
> 
> The Upload/Download section of https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=3588%3A800 is probably the simplest view of it—with https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=4178%3A1794 going into a lot more depth on various edge cases.

Ok so all of this has already been implemented in the code?  If so, that works for me. For some reason I just thought there was more for V2. I think once Figma is cleaned up some more (organized in the correct states) things will be more clear.

**Comment by reitermb** [comment]
> > > @reitermb is there documentation for the current submission portion?
> > 
> > 
> > Yeah, that was on the list of past (v0/ATO) release spec we'd discussed not adding to the repo last week, but I did dig up the right Figma pages for it.
> > The Upload/Download section of https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=3588%3A800 is probably the simplest view of it—with https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=4178%3A1794 going into a lot more depth on various edge cases.
> 
> Ok so all of this has already been implemented in the code? If so, that works for me. For some reason I just thought there was more for V2. I think once Figma is cleaned up some more (organized in the correct states) things will be more clear.

Correct! Notifications and onboarding are the two remaining UX lifts for V2—the former will be entirely content driven (and delivered in HTML & hack.md), the former may involve _some_ Figma work, but will also be largely content oriented. So currently the only non-developed Figma spec for V2 is Submission History.

**Review by lfrohlich — APPROVED** [review]
LGTM

File: docs/User-Experience/Dev-Ready-Designs/README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # Designs for Dev
+This folder stores dev-ready designs so that dev tickets can point to a static and updated location for design spec rather than potentially living/changing documents in Figma. 
 
-_forthcoming_
\ No newline at end of file
+## V2 Designs
+- [Submission History V2](https://github.com/raft-tech/TANF-app/blob/docs/User-Experience/Dev-Ready-Designs/Submission%20History%20V2.pdf)
```
Comment (ADPennington) [inline docs/User-Experience/Dev-Ready-Designs/README.md:5]: could you add a brief blurb on what is represented here.

File: docs/User-Experience/Dev-Ready-Designs/README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+## V3 Designs
+- [Submission History V3](https://github.com/raft-tech/TANF-app/blob/docs/User-Experience/Dev-Ready-Designs/Submission%20History%20V3.pdf)
```
Comment (ADPennington) [inline docs/User-Experience/Dev-Ready-Designs/README.md:8]: same suggestion as above @reitermb

**Review by ADPennington — CHANGES_REQUESTED** [review]
minor suggestions recommended.

**Review by stevenino — APPROVED** [review]

**Review by abottoms-coder — APPROVED** [review]

**Pull request merged**